core/ops_builtin_v8.rs

```diff
@@ -1047,9 +1047,10 @@ fn write_line_and_col_to_ret_buf(
 // 2: mapped line, column, and file name. new line, column, and file name are in
 //    ret_buf. retrieve file name by calling `op_apply_source_map_filename`
 //    immediately after this op returns.
-#[op2(fast)]
+#[op2]
 #[smi]
 pub fn op_apply_source_map(
+  scope: &mut v8::HandleScope,
```

Comment:
This should be reverted and a fallback op - `op_apply_source_map_slow` should be added. Then the callsite should fallback to the slower op if this one returns "unchanged" application.

Until we get more confident in this functionality I think we should still use fast version that goes through the `SourceMapGetter` implementation first.

Thoughts @lucacasonato?

Reply:
Discussed offline, we're not gonna add a fallback op. With upcoming changes to V8 in 12.8 this will still be eligible to be a fast op.